rules/codelingo/effective-go/initialize-using-composite-literal/codelingo.yaml

```
@@ -4,9 +4,38 @@ tenets:
       codelingo/docs:
         title: Initialize instance using composite literal
         body: |
-          Sometimes the zero value isn't good enough and an initializing constructor is necessary. We can simplify the code using a composite literal, which is an expression that creates a new instance each time it is evaluated.
+          Sometimes the zero value isn't good enough and an initializing constructor 
+          is necessary. We can simplify the code using a composite literal, which is 
+          an expression that creates a new instance each time it is evaluated.
+
+          ## Bad code
+          ```
+          //NewFile is a function containing an initializing constructor. 
```

```suggestion
          // NewFile is a function containing an initializing constructor. 
```
Please always put a space after //